crates/nu-command/tests/commands/str_/mod.rs

```diff
@@ -96,6 +96,7 @@ fn upcases() {
     })
 }
 
+#[ignore]
```

Comment:
i would add a reason for this ignore :yum: 
```suggestion
#[ignore = "Playgrounds are not supported in nu-cmd-extra"]
```